com.unity.multiplayer.mlapi/Tests/Runtime/Metrics/NetworkObjectMetricsTests.cs

```diff
@@ -1,3 +1,4 @@
+#if MULTIPLAYER_TOOLS
```

Comment:
The consensus we came up with @mattwalsh-unity is that we want tests to break if the tools are not present.

Reply:
Oh right, I remember this. So this PR should actually be to add the tools package to the test project (now that it's in the internal registry)

Reply:
Actually - I'd like to discuss this again. I'll move the discussion to slack.